### ISSUE TITLE
Lazy loaded route displayed an empty page

### ISSUE THREAD

**Issue by ukrukarg** [angular/angular#24611]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
[ ] Other... Please describe:
</code></pre>

## Current behavior
In my project we lazy-load a lot and take advantage of the fact that you can lazily load multiple modules for the same route.

I load 2 modules for route 'admin',
1st module:
```
 const adminRoutes: Routes = [
  {
    path: 'admin2',
    component: AdminComponent,
  }
];
```

2nd module:
```
const adminRoutes: Routes = [
  {
    path: '',
    component: AdminComponent,
    children: [
      {
        path: '',
        children: [
          { path: 'crises', component: ManageCrisesComponent },
          { path: 'heroes', component: ManageHeroesComponent },
          { path: '', component: AdminDashboardComponent }
        ]
      }
    ]
  }
];
```
When I navigate to `/admin/admin2` or `/admin/crises` everything loads correctly.

When I navigate to `/admin` blank page is displayed.

## Expected behavior
Navigating to `/admin` displays admin page.

## Minimal reproduction of the problem with instructions
https://stackblitz.com/edit/angular-y1xxzq?file=src/app/app-routing.module.ts

Steps to repro:
1. Click on Admin link -> see empty page
2. Directly navigate to `/admin/admin2` or `/admin/crises` -> works

Changing the order of routes in `src/app/app-routing.module.ts` seems to fix the issue.

## What is the motivation / use case for changing the behavior?
The way we build routes in my app is very dynamic, I can't necessarily enforce the order of the routes.

**Comment by alex-okrushko** [comment]
`hotlist: google` as well

**Comment by ukrukarg** [comment]
FYI I managed to force order of my routes, so unblocked at the moment.

**Comment by ukrukarg** [comment]
I basically ordered by paths in RoutingModule in a way that makes it work. 
In the example: if you flip the order of routes in https://stackblitz.com/edit/angular-y1xxzq?file=src/app/app-routing.module.ts to be this:
```
  {
    path: 'admin',
    loadChildren: 'app/admin/admin.module#AdminModule',
  },
 {
    path: 'admin',
    loadChildren: 'app/admin2/admin2.module#Admin2Module',
  },
```
than this makes it work.

**Comment by agale123** [comment]
Google-internal bug tracking this is b/130235987

**Comment by yueluog** [comment]
Found a related bug: https://github.com/angular/angular/issues/12904

**Comment by atscott** [comment]
Closing as duplicate of #12904
